doc/architecture/decisions/0008-concept-of-bedspaces.md

```diff
@@ -0,0 +1,62 @@
+# 8. concept-of-bedspaces
+
+Date: 2023-02-28
+
+## Status
+
+Accepted
+
+## Context
+
+CAS1 and CAS3 share an API which includes the modelling for premises, rooms and
+beds. This modelling was initially set out by CAS1 and CAS3 have been working to
+extend in order to meet the needs of Temporary Accommodation (CAS3).
+
+The relationships are straight forward. A premises can have many rooms and a
+room can have many beds.
+
+For CAS1 they will often have the scenario of multiple beds belonging to a single
+room.
+
+For CAS3 we only expect one bed per room for the foreseeable future however
+cannot guarantee that this will always remain the case.
+
+At the time of writing an individual bed record doesn't itself contain any data,
+this too may change in future.
+
+Rooms serve to group related beds and to encapsulate shared attributes such as
+physical characteristics of the space. These are used in the matching journey
+where the risks and needs of person leaving prison are matched up to a suitable
+premises and room.
+
+Both teams have the need to help users create bookings for physical beds and to
+report on occupancy levels based on bed, rather than room. This points us
+towards using the existing bed model as the smallest unit of data for both
+services.
+
+Given the CAS3 team have 1 bed per room, we wanted to tailor the user
+experience to avoid unnecessary clicks. Instead of asking the user to navigate
+from premises through to rooms and on to the bed we can instead present all beds
+in a premises alongside their room characteristics. This removes the concept of
+'rooms' from our user.
+
+Finally, our users were more familiar with the language of 'bedspace' over 'bed'.
+
+## Decision
+
+Temporary Accommodation will introduce the concept of a "bedspace" to users.
+
+## Consequences
```

Comment:
Suggest start each of these with a capital letter

Reply:
At some point my brain seems to have formed a habit of following [the GDS design system for bulleted lists](https://design-system.service.gov.uk/styles/typography/#bulleted-lists)! There doesn't seem to be any style guide for markdown but so I think it makes sense to do as you say 👍